agagd/templates/static_pages/information.html

```diff
@@ -26,7 +26,25 @@ <h2>Additional Information</h2>
     <li>Answers to the <a href="https://www.usgo.org/files/pdf/ratingquestions.pdf">most commonly asked questions about the rating system</a> (May 1992)</li>
     <li>A more detailed <a href="https://www.usgo.org/files/pdf/AGARatings-Math.pdf">mathematical description</a> of the ratings system (April 2010)</li>
     <li><a href="https://www.usgo.org/files/pdf/bayrate.zip">Source code</a> for the AGA rating algorithm</li>
-    <li>Tournament Directors: <a href="https://usgo.org/mm/tdlista.txt">TDListA</a>  <a href="https://usgo.org/mm/tdlistn.txt">TDListN</a></li>
+    <li>Tournament Directors: <br>
```

Comment:
We would probably do better moving the title into an `<h3></h3>` or another header; then moving the table beneath that and having the label "TDLists Formating." May be change the label for "Additional information" to "Resources."

Reply:
The page probably needs an overhaul of info anyways. Next update?

Reply:
Yes. We could just move this forward, and on the next update the page can be overhauled.